client/stylesheets/_toCat.scss

```diff
@@ -3753,6 +3754,7 @@ On click the images will be displayed at normal size to complete the effect
 .box-title {
     padding: 5px 8px;
     font-size: 1.1rem;
+    font-weight: 400 !important;
```

Comment:
!important should not be used<br>Line should be indented 2 spaces, but was indented 4 spaces